### PR TITLE
Adds new brick blocks

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Adds new blocks, slabs, stairs, and walls for brown, beige, and cream textures
- Textures from Shindria, blocks requested by LdW
- Tested and works 🎉
- (probably too many) screenshots below:

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/structurize/blocks/bricks/BlockBricks.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.ldtteam.structurize.blocks.bricks;
+
+import com.ldtteam.structurize.blocks.AbstractBlockStructurize;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+
+public class BlockBricks extends AbstractBlockStructurize<BlockBricks>
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structurize/blocks/bricks/BlockBricks.java:12]: Javadoc and newline (same for the other ones)

**Comment by Raycoms** [comment]
Just missing some javadoc

File: src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+public abstract class AbstractBlockStructurizeWall<B extends AbstractBlockStructurizeWall<B>> extends WallBlock implements IBlockStructurize<B>
+        /**
+         * Extend the vanilla WallBlock
+         */
+{
+    public AbstractBlockStructurizeWall(final Block.Properties properties)
+    {
+        /**
+         * Lets you set the block properties
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java]: no need for doc of this super call

File: src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+import net.minecraftforge.registries.IForgeRegistry;
+
+/**
+ * The abstract class for structurize-added walls.
+ * Currently only used by brown brick, beige brick,
+ * and cream brick walls.
+ */
+
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java:15]: this new line is not necessary

File: src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+public abstract class AbstractBlockStructurizeWall<B extends AbstractBlockStructurizeWall<B>> extends WallBlock implements IBlockStructurize<B>
+        /**
+         * Extend the vanilla WallBlock
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java]: this javadoc comment is "in the middle of nowwhere"

There should probably be one above the constructor, and with an @parameter

File: .gitignore [hunk 5]
```diff
@@ -28,3 +28,5 @@
 /libs
 
 /src/main/generated/java
+
+logs/
```
Comment (Nightenom) [inline .gitignore]: Should be `/logs/`, also I have never seen this folder so not sure how you manage to get it :D

Comment (Raycoms) [reply .gitignore]: I got those too

Comment (Nightenom) [reply .gitignore]: Weird..

**Review by Nightenom — APPROVED** [review]

**Pull request merged**